google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/apis/AddCloudLibrariesWizardAction.java

```diff
@@ -16,6 +16,7 @@
 
```

Comment:
General comment about this class - it starts feeling a bit confusing for me - it's an action, inside of the action is a Wizard, as a nested class, then another nested class is a wizard step, and this steps returns actual UI, which is a separate class.  Could we refactor a bit later - for example, separate action class, remove wizard step and combine wizard and its only step as one simple dialog?

Reply:
yes. that is a todo I've had. I'll remove the wizard in a separate PR